src/test/java/com/google/javascript/clutz/partial/global_generic_type.js

```diff
@@ -0,0 +1,4 @@
+goog.provide('global.generic.type');
+
+/** @type {Map<string, string>} */
+global.generic.type = null;
```

Comment:
Can you add a test where it uses Map without template params?

Reply:
Done!